src/pages/iou/MoneyRequestEditWaypointPage.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import WaypointEditor from './WaypointEditor';
+
+const propTypes = {
+    /** The transactionID of this request */
+    transactionID: PropTypes.string,
+
+    /** Route params */
+    route: PropTypes.shape({
+        params: PropTypes.shape({
+            /** Thread reportID */
+            threadReportID: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+
+            /** ID of the transaction being edited */
+            transactionID: PropTypes.string,
+
+            /** Index of the waypoint being edited */
+            waypointIndex: PropTypes.string,
+        }),
+    }),
+};
+
+const defaultProps = {
+    route: {},
+};
+
+// This component is responsible for grabbing the transactionID from the IOU key
```

Comment:
Is it some magic I'm not familiar with? This component seems to do nothing beyond forwarding the `route` property

Reply:
Or was this comment copy-pasted?